server/controller/accountController.js

```diff
@@ -16,22 +17,24 @@ class AccountController {
       email, firstName, lastName,
     } = req.customer;
     const { type, id } = req.body;
-    if (type === 'savings' || type === 'current') {
+    const validate = RegularExpression.validate();
+    if (validate.accountType.test(type)) {
       const accountNumber = AccountGenerator.accountGenerator();
       const newAccount = {
-        id: accounts.length + 1,
+        // id: accounts.length + 1,
         accountNumber,
         createdOn: new Date(),
         owner: id,
         type,
         status: 'draft',
         balance: 0,
       };
-
-
       const createdOn = new Date().getTime();
+      // const accounts = [];
+      // accounts[accounts.length] = 'newAccounts';
```

Comment:
This should not be here. I expected that the feedback on the API endpoints should have been implemented on another PR.